### ISSUE TITLE
Crash on Spanish (es_ES) Locale

### ISSUE THREAD

**Issue by alexcontor** [slgobinath/SafeEyes#306]
Hi , im getting crashes eventually on Ubuntu 18.04.2 LTS : here is the log:

safeeyes crashed with TypeError in__set_next_break_info(); not all arguments are converted during string formatting.

I tried disabling samrtbreak plugin but this error still persists. Any hint?

**Comment by slgobinath** [comment]
Hi,
It happens if your locale language is not translated completely. Could you please share your locale?
Entering `locale` in the terminal will print your locales.

**Comment by alexcontor** [comment]
@slgobinath 
LANG=es_ES.UTF-8
LANGUAGE=
LC_CTYPE="es_ES.UTF-8"
LC_NUMERIC="es_ES.UTF-8"
LC_TIME="es_ES.UTF-8"
LC_COLLATE="es_ES.UTF-8"
LC_MONETARY="es_ES.UTF-8"
LC_MESSAGES="es_ES.UTF-8"
LC_PAPER="es_ES.UTF-8"
LC_NAME="es_ES.UTF-8"
LC_ADDRESS="es_ES.UTF-8"
LC_TELEPHONE="es_ES.UTF-8"
LC_MEASUREMENT="es_ES.UTF-8"
LC_IDENTIFICATION="es_ES.UTF-8"
LC_ALL=

**Comment by slgobinath** [comment]
Hi,

There is an error in the translation at https://github.com/slgobinath/SafeEyes/blob/master/safeeyes/config/locale/es/LC_MESSAGES/safeeyes.po#L435

```
msgid "Next break at %s"
msgstr "Siguiente descanso en"
```

`%s` is missing in the translation. If you disable the tray icon you should be fine. Could you please share the correct translation.

**Comment by alexcontor** [comment]
Sure! The translation of "Next break at.." in spanish would be : "El siguiente descanso será en" ... or if you want it abreviated : "Siguiente descanso en..."

**Comment by slgobinath** [comment]
> PD: I disabled the tray icon and the error still persists.

That is weird. I will try my best to fix this problem as early as possible.

Is `El siguiente descanso será en 12.00` is equivalent to `Next break at 12.00`?

**Comment by alexcontor** [comment]
Yeah , it is.

Thanks :)

**Comment by slgobinath** [comment]
Fixed with https://github.com/slgobinath/SafeEyes/commit/6927ca4500e655df7d60574e124da3232a10f920
Could you please check if the following folder exist:
```
ls /usr/lib/python3/dist-packages/safeeyes/config/locale/es/LC_MESSAGES/
```

If it is there, please download and extract the attached file and replace `/usr/lib/python3/dist-packages/safeeyes/config/locale/es/LC_MESSAGES/safeeyes.po` and `/usr/lib/python3/dist-packages/safeeyes/config/locale/es/LC_MESSAGES/safeeyes.mo` files.

Finally restart the Safe Eyes and see if it works.

Thanks
[es_LC_MESSAGES.zip](https://github.com/slgobinath/SafeEyes/files/2914894/es_LC_MESSAGES.zip)

**Comment by slgobinath** [comment]
Fixed in 2.0.9
